Sampler/Templates/Sampler/plasterManifest.xml

```diff
@@ -491,7 +488,7 @@
    <!-- CHANGELOG MD -->
    <file source='CHANGELOG.md.template'
```

Comment:
With the comment above using `$PLASTER_PARAM_ModuleName` we should change the type from `<file ` to `<templateFile `.

Reply:
Done